pkgs/os-specific/darwin/apple-source-releases/default.nix

```diff
@@ -1,11 +1,14 @@
 { stdenv, fetchurl, fetchzip, pkgs }:
 
 let
+  macosPackages = import ./macos-11.0.1.nix { inherit applePackage; };
+  developerToolsPackages = import ./developer-tools-11.3.1.nix { inherit applePackage; };
+
   # This attrset can in theory be computed automatically, but for that to work nicely we need
   # import-from-derivation to work properly. Currently it's rather ugly when we try to bootstrap
   # a stdenv out of something like this. With some care we can probably get rid of this, but for
   # now it's staying here.
-  versions = {
+  sdkNames = {
```

Comment:
This map is really about versions, not "names" of Apple SDKs.

Reply:
`versions` may cause some confusion between packages version and sdk version. Below we are using `version = sdkNames.${sdkName}.${pname} or "";`, if changing to `versions.${sdkName}` would it be a little strange?

Reply:
`packageVersion`?